examples/async_tasks/async_compute.rs

```diff
@@ -104,10 +104,10 @@ fn spawn_tasks(mut commands: Commands) {
     }
 }
 
-/// This system queries for entities that have our Task<Transform> component. It polls the
-/// tasks to see if they're complete. If the task is complete it takes the result, adds a
-/// new [`Mesh3d`] and [`MeshMaterial3d`] to the entity using the result from the task's work, and
-/// removes the task component from the entity.
+/// This system queries for entities that have our Task<CommandQueue> component. It polls the
+/// tasks to see if they're complete. If the task is complete it takes the result, and appends
+/// all the commands that were queued to the command object, so they will actually run.
+/// No application specific code is needed here, so this system could be in a separate plugin.
```

Comment:
The component used is `ComputeTransform`, `Tasks` are not components.

This also means that the last line you added is wrong